scv-api/obj/Debug/netcoreapp3.1/scv-api.AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+//------------------------------------------------------------------------------
```

Comment:
Remove all artifacts (`**/obj/**`) from the repository, and add a `.gitignore` file to ensure artifacts don't get checked into source control.  A collection of standard gitignore examples can be found here.